src/inquirer/render/console/_checkbox.py

```diff
@@ -96,6 +96,12 @@ def process_input(self, pressed):
         elif pressed == key.RIGHT:
             if self.current not in self.selection:
                 self.selection.append(self.current)
+        elif pressed == key.CTRL_A:
+            self.selection = [i for i in range(len(self.question.choices))]
```

Comment:
```suggestion
            self.selection = [i for i, _ in enumerate(self.question.choices)]
```

its not much better, but one less parentisees and more on point wiht whats actually happening